src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+interface UserInfoProbs {
+  user: User | null,
```

Comment:
If user was null this component wouldn't even render, you can also remove ? from the properties
```suggestion
  user: User,
```